app/services/waste_carriers_engine/analytics/user_journey_service.rb

```diff
@@ -27,7 +27,7 @@ def run(transient_registration:, current_user: nil)
         @current_user = current_user
         @journey = find_or_create_user_journey(journey_type, token)
 
-        PageView.create!(page: page, time: Time.zone.now, route: route, user_journey: journey)
+        journey.page_views.create!(page:, route:, time: Time.zone.now)
```

Comment:
Should it be @journey.page_views.create! instead? I see `@journey` variable being set, but not `journey`

Reply:
There's an `attr_accessor :journey` so `journey` also works here.

Reply:
ok